git/00.setup.md

```diff
@@ -42,8 +42,8 @@ combinations.
 ```bash
 git config --global alias.aa add --all
 git config --global alias.d diff --staged
-git config --global alias.count shortlog -sn
-git config --global alias.l log --pretty=format:%C(yellow)%h%Cblue%d%Creset %s %C(white) %an, %ar%Creset --graph
+git config --global alias.pf push --force-with-lease
+git config --global alias.l log --graph --decorate --oneline
```

Comment:
OK. I'm fine to keep yours. Mine looked more useful to me at a glance, but if this is something that you use every day then I could very well be wrong. :-) I took yours out speculatively, guessing that there might not be a strong attachment there, but it doesn't matter greatly to me. Could even have two aliases, like `l1` and `l2` or something. Whatever.